From 9a34a5be5e27628737cff28ef3697ea2e0dca8d5 Mon Sep 17 00:00:00 2001
From: Alexey Brodkin <Alexey.Brodkin@synopsys.com>
Date: Thu, 7 Feb 2013 18:26:15 +0400
Subject: [PATCH 0726/1852] xilinx: Documentation/devicetree - add description
 for "port-number" property

commit 353cfd73fca880ea4a0740ffd6b7726a1f6fd30c from
https://github.com/Xilinx/linux-xlnx.git

This property is used by xsysace device (drivers/block/xsysace.c).

Signed-off-by: Alexey Brodkin <abrodkin@synopsys.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/xilinx.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/xilinx.txt b/Documentation/devicetree/bindings/xilinx.txt
index d058ace29345..0c75bb153ca6 100644
--- a/Documentation/devicetree/bindings/xilinx.txt
+++ b/Documentation/devicetree/bindings/xilinx.txt
@@ -253,6 +253,7 @@
 
       Optional properties:
        - 8-bit (empty) : Set this property for SystemACE in 8 bit mode
+       - port-number = <port_number> : Set port number for particular device
 
       iii) Xilinx EMAC and Xilinx TEMAC
 
-- 
2.31.1

